x-pack/legacy/plugins/code/index.ts

```diff
@@ -31,11 +31,11 @@ export const code = (kibana: any) =>
     publicDir: resolve(__dirname, 'public'),
 
     uiExports: {
```

Comment:
how to migrate all the items in `uiExports` to new plugin?

Reply:
-  `app` - superseded by Application service
- `styleSheetPaths` keep in the legacy platform for a while. Should be done in https://github.com/elastic/kibana/issues/42185
- `injectDefaultVars` keep there as well. Can be re-visited soon https://github.com/elastic/kibana/issues/41990
- `hacks` move this logic in the `setup` phase of the client-side new platform plugin. The new platform runs all plugin, so this behavior replicates legacy `hacks`.